Services/CO.CDP.Organisation.WebApi/Program.cs

```diff
@@ -83,9 +83,9 @@
 builder.Services.AddOrganisationProblemDetails();
 
 builder.Services.AddJwtBearerAndApiKeyAuthentication(builder.Configuration, builder.Environment);
-//builder.Services.AddAuthorization();
-builder.Services.AddOrganisationAuthorization();
-builder.Services.AddGovUKNotifyApiClient(builder.Configuration);
+builder.Services.AddAuthorization();
+//builder.Services.AddOrganisationAuthorization();
+//builder.Services.AddGovUKNotifyApiClient(builder.Configuration);
```

Comment:
Probably want to revert these changes before they go to master

Reply:
good spotted, resolved